Runtime/Cesium3DTileset.cs

```diff
@@ -70,6 +70,16 @@ internal static void
             }
         }
 
+        public event Action<GameObject> OnNewGameObjectCreated;
```

Comment:
Add comments for this event, similar to the others. Something like this:
```
/// <summary>
/// An event that is raised when a new tile game object is created for the tileset.
/// This can be used to modify tiles as they are loaded -- for example, adding additional components.
/// </summary>
```